flutter/lib/resources/archive_cache_helper.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:io';
+
+import 'package:archive/archive_io.dart';
+
+import 'file_cache_helper.dart';
+
+class ArchiveCacheHelper {
+  static const extension = '.zip';
+  final FileCacheHelper cacheManager;
```

Comment:
The variable is named as `cacheManager` but has type `FileCacheHelper` and not `CacheManager`. Should be `fileCacheHelper`.

Reply:
fixed